x-pack/plugins/maps/public/components/widget_overlay/layer_control/layer_toc/toc_entry/_toc_entry.scss

```diff
@@ -1,6 +1,7 @@
 .mapTocEntry {
   padding: $euiSizeS $euiSize;
   position: relative;
+  background-color: $euiColorEmptyShade;
```

Comment:
is it me, or does this green look a little odd on top of the map? the map background can have all kinds of colors, so this lightgreen looks odd (e.g. on top of the blue of water)

Reply:
At this moment, that's not alterable outside of EUI. It goes along with my comment altering some styling on a follow-up. But that color is the same as it will be for everyone.